### PR TITLE
Add scenario 16 - Deploy multiple swagger-projects as one API

### PR DESCRIPTION
**Pull request by renuka-fernando**
**Create API**

```sh
>> apictl add api -n petstore-multiple-api --from-file=pets-int --from-file=stores-int 
```

**Invoke Pets API**
    
```sh
curl -X GET "https://<external IP of LB service>:9095/pet/v1/pet/10" -H "accept: application/json" -H "Authorization:Bearer $TOKEN" -k
```

Output:

```json
{"id":10,"category":{"id":10,"name":"dolor"},"name":"eiusmod","photoUrls":["${photoUrls}","${photoUrls}"],"tags":[{"id":10000,"name":"Lorem"},{"id":10000,"name":"Lorem"}],"status":"consectetur"}
```

**Invoke Stores API**

```sh
curl -X GET "https://<external IP of LB service>:9095/store/v1/store/inventory" -H "accept: application/json" -H "Authorization:Bearer $TOKEN" -k
```

Output:

```json
{"incididunt":4,"tempor":6,"string":626,"pending":19,"adipiscing":6,"available":109,"do":6,"dolor":4,"freaky":2,"sed":4,"scary":1,"sit":10,"ut":8,"sold":8,"labore":2,"eiusmod":10,"magna":6,"et":4,"0":5,"dolore":6,"for sale":1,"Lorem":4,"amet":8,"ipsum":10,"elit":2,"consectetur":8}
```

### CODE REVIEW

File: scenarios/scenario-16/stores_swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          type: string
+          description: Order Status
+          enum:
+            - placed
+            - approved
+            - delivered
+        complete:
+          type: boolean
+      xml:
+        name: Order
```
Comment (dinusha92) [inline scenarios/scenario-16/stores_swagger.yaml]: Add a new line :)

File: scenarios/scenario-16/pets_swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    Pet:
+      content:
+        application/json:
+          schema:
+            "$ref": "#/components/schemas/Pet"
+        application/xml:
+          schema:
+            "$ref": "#/components/schemas/Pet"
+      description: Pet object that needs to be added to the store
+      required: true
```
Comment (dinusha92) [inline scenarios/scenario-16/pets_swagger.yaml]: Shall we add a new line here

File: scenarios/scenario-16/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+- This scenario describes how to deploy the petstore service(https://petstore.swagger.io/v2) on a kubernetes cluster as a managed API.
+
+ ***Important:***
+> Follow the main README and deploy the apim-operator and configuration files. Make sure to set the analyticsEnabled to "true" and deploy analytics secret with credentials to analytics server and certificate, if you want to check analytics.
+
+##### Deploying the artifacts
+
+- Navigate to scenarios/scenario-16 directory.
+- Prepared petstore swagger definitions `pets_swagger.yaml` and `stores_swagger.yaml` can be found within this directory.
+- Base paths of the APIs are mentioned in the swagger files with the  "x-wso2-basePath".
```
Comment (dinusha92) [inline scenarios/scenario-16/README.md]: last part -> with the "x-wso2-basetpath" **vendor extension**.

File: scenarios/scenario-16/pets_swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,180 @@
+#Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Meruja) [inline scenarios/scenario-16/pets_swagger.yaml:1]: Shall we add the copyright year?

File: scenarios/scenario-16/stores_swagger.yaml [hunk 5]
```diff
@@ -0,0 +1,111 @@
+#Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Meruja) [inline scenarios/scenario-16/stores_swagger.yaml:1]: missing year

**Pull request merged**